dataflow/src/main/java/org/checkerframework/dataflow/cfg/DOTCFGVisualizer.java

```diff
@@ -206,21 +206,14 @@ protected String dotOutputFileName(UnderlyingAST ast) {
         return outFileName;
     }
 
+    @Override
+    protected String format(Object obj) {
+        return escapeDoubleQuotes(obj);
+    }
+
     @Override
     public String visualizeBlockNode(Node t, @Nullable Analysis<V, S, T> analysis) {
```

Comment:
Now that the two `visualizeBlockNode` methods are exactly the same, should we just remove them and make the `Helper` the real implementation in `AbstractCFGVisualizer`?